packages/lib-user/src/components/GroupAllProjects/GroupAllProjects.js

```diff
@@ -69,8 +43,33 @@ function GroupAllProjects({ authUser, group }) {
         />
       }
     >
-      <ContentBox title='All Projects' pad='45px'>
-        <AllProjects error={error} loading={loading} projects={topProjects} />
+      <ContentBox pad='45px'>
+        <Heading level='1'>
```

Comment:
This is my "on the fly" decision to include some actual headers on this page even though they're missing from Figma. I'll consult with Sean about what'd he like these to look like.